src/attributecode/attrib.py

```diff
@@ -193,8 +193,19 @@ def generate_sctk_input(abouts, min_license_score, license_dict):
                 for key in lic_key:
                     lic_name.append(license_dict[key][0])
             lic_score = about.license_score.value
-            assert len(lic_key) == len(lic_name)
-            assert len(lic_key) == len(lic_score)
+            if len(lic_key) != len(lic_name):
```

Comment:
May be you can extract a variable?
```suggestion
            len_lics = len(lic_key)
            if len_lics != len(lic_name):
```

and then use it in the messages and other tests below?